Libs/Gtk3/Classes/Widget.CompositeWidget.cs

```diff
@@ -57,21 +57,21 @@ protected static void BindTemplateSignals(Type gtype, System.Type t)
         private static void OnConnectEvent(IntPtr builder, IntPtr @object, string signal_name, string handler_name,
             IntPtr connect_object, ConnectFlags flags, IntPtr user_data)
         {
-            if(!TryWrapPointerAs<Widget>(@object, out var eventSource))
+            if(!TryWrapHandle<Widget>(@object, out var eventSource))
                 return;
 
-            if(!TryWrapPointerAs<Widget>(connect_object, out var compositeWidget))
+            if(!TryWrapHandle<Widget>(connect_object, out var compositeWidget))
                 return;
 
-            MethodInfo? compositeWidgetCallbackMethodInfo = compositeWidget.GetType().GetMethod(
+            MethodInfo? compositeWidgetCallbackMethodInfo = compositeWidget?.GetType().GetMethod(
```

Comment:
The question mark should not be needed if `TryWrapHandle` is fixed with the correct attribute (see corresponding comment).

Reply:
Fixed